src/main.rs

```diff
@@ -119,7 +169,8 @@ fn main() -> Result<(), ureq::Error> {
     Ok(())
 }
 
-fn print_item(query: &str, value: &Value) -> Option<()> {
+fn print_item(query: &str, value: &Value, output: &mut String) -> Option<usize> {
+    let mut aux;
```

Comment:
Why declaring aux without value instead of simply using `let aux = ` in line 183?